src/lwc/gsChecklistItem/gsChecklistItem.css

```diff
@@ -0,0 +1,121 @@
+.item-card {
+    position: relative;
+    min-height: 185px;
+    border: 1px solid lightgray;
+    width: 100%;
+    margin: 1.5rem 0;
+    padding: 18px;
+    box-shadow: 0 3px 2px 0 rgba(176,173,171,.48);
+    display: grid;
+    grid-template-columns: 185px auto;
+}
+
+.item-card .card-image {
+    width: 185px;
+    grid-column: 1/ 2;
+    padding-left: 6px;
+    padding-right: 24px;
+    border-right: 1px solid lightgray;
+}
+
+.item-card .card-image img {
+    width: 100%;
+}
+
+.item-card .card-content {
+    display: flex;
+    position: relative;
+    grid-column: 2/ 3;
+    padding-left: 14px;
+    flex-direction: column;
+    justify-content: space-between;
+}
+
+.item-card .card-content .content-time {
+    position: absolute;
+    right: 0;
+    height: 14px;
+    width: 266.91px;
+    color: #706E6B;
+    font-family: "Salesforce Sans";
+    font-size: 12.98px;
+    letter-spacing: 0;
+    line-height: 19.97px;
+    text-align: right;
+}
+
+.item-card .card-content .content-title {
+    color: #3E3E3C;
+    font-family: "Salesforce Sans";
```

Comment:
@bethbrains What's your opinion on specifying font and color in a custom CSS versus using standard SLDS designs?

Reply:
We had a tons of conversations regarding this. Colors schema was provided by UX/UI designer and  all colors were validated prior we started (I remember talking about it with Kennen and Stephanie).